_config.yml

```diff
@@ -20,6 +20,15 @@ baseurl: /al-folio # the subpath of your site, e.g. /blog/
 last_updated: false # set to true if you want to display last updated in the footer
 impressum_path:  # set to path to include impressum link in the footer, use the same path as permalink in a page, helps to conform with EU GDPR
 
+# -----------------------------------------------------------------------------
+# RSS Feed
+# -----------------------------------------------------------------------------
+# will use title and url fields
+description:  # Higly suggested: FILL THIS FIELD! If blank, this could lead to errors. A longer description of what your site is about, e.g., "Where I blog about Jekyll and other awesome things"
+author:  # your name and surname
+feed:
+  excerpt_only: false # set it to have all posts in feed will be without <content> tags.
```

Comment:
These fields seem to be optional. I'd prefer to remove these fields and refer users to `jekyll-feed` docs instead, since there many more options. Basically, simply substitute all fields in this subsection with a brief comment with a link to https://github.com/jekyll/jekyll-feed.